businesskey-import-plugin/anonymize-businesskeys/src/main/java/org.camunda.optimize.examples/AnonymizeBusinessKeys.java

```diff
@@ -0,0 +1,19 @@
+package org.camunda.optimize.examples;
+
+import org.camunda.optimize.plugin.importing.businesskey.ProcessInstancePluginDto;
+import org.camunda.optimize.plugin.importing.businesskey.BusinessKeyImportAdapter;
+import java.util.List;
+
+public class AnonymizeBusinessKeys implements BusinessKeyImportAdapter {
+
+  public List<ProcessInstancePluginDto> adaptBusinessKeys(List<ProcessInstancePluginDto> processInstances) {
+    processInstances.stream()
+      .forEach(this::anonymizeBusinessKey);
+    return list;
+  }
+
+  private void anonymizeBusinessKey(ProcessInstancePluginDto processInstance) {
+    final int anonymizedBusinessKey = processInstance.getBusinessKey().hashCode();
```

Comment:
just to be sure, let's maybe add a comment here that his is just a simple sample of converting the key to a hash but generally more state-of-the-art secure hashing algorithms should be used?